### PR TITLE
Some basic files of protocol  SpdzWise in field

### PR DESCRIPTION
**Pull request by DuanYuFi**
I want to add the SPDZ-wise protocol for computation modulo a prime by referring [Chida et al.](https://eprint.iacr.org/2018/570). And now as the file structure of spu changes recently, I want to firstly PR some basic files and fill in the details later. 
I also referred what [MP-SPDZ](https://github.com/data61/MP-SPDZ/blob/master/Protocols/SpdzWiseShare.h) done, so it is not fully follows by the paper.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -54,3 +54,4 @@ idir/
 
 spu_wheel.name
 *.whl
+.vscode/settings.json
```
Comment (anakinxc) [inline .gitignore:57]: revert this change please

**Review by anakinxc — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (DuanYuFi) [reply .gitignore]: OK, done.

Comment (anakinxc) [reply .gitignore]: Can you also add settings.json back?

Comment (DuanYuFi) [reply .gitignore]: Is that OK now? I'm not very familiar with git yet :(

Comment (anakinxc) [reply .gitignore]: perfect :P

This change looks fine to me. Let's wait for feedbacks from other reviewers.

Comment (DuanYuFi) [reply .gitignore]: Thank you :)
